lib/tasks/webpacker/check_binstubs.rake

```diff
@@ -1,7 +1,7 @@
 namespace :webpacker do
   desc "Verifies that webpack & webpack-dev-server are present."
   task :check_binstubs do
-    unless Bundler.which(Gem.win_platform? ? "webpack.bat" : "webpack")
+    unless File.exist?("bin/webpack") && File.exist?("bin/webpack-dev-server")
```

Comment:
@rossta I think it's fine to leave out check for dev server since people might like to .gitignore it for production.

Reply:
Ah, right, I was reverting to older behavior.